src/Spd.Resource.Repository/BizLicApplication/BizLicApplicationRepository.cs

```diff
@@ -20,7 +20,6 @@ public async Task<BizLicApplicationResp> GetBizLicApplicationAsync(Guid licenceA
     {
         spd_application? app = await _context.spd_applications
             .Expand(a => a.spd_ServiceTypeId)
-            .Expand(a => a.spd_ApplicantId_account)
```

Comment:
biz app is connected with account, why we delete it here while keeping contact?

Reply:
I was using that extension to get the biz id, instead of that now I get it from organization id from the `spd_applications` table